infrastructure/stacks/development-pipeline/unit-tests-buildspec.yml

```diff
@@ -0,0 +1,65 @@
+version: 0.2
+
+#env:
+  #variables:
+    # key: "value"
+    # key: "value"
+  #parameter-store:
+    # key: "value"
+    # key: "value"
+  #secrets-manager:
+    # key: secret-id:json-key:version-stage:version-id
+    # key: secret-id:json-key:version-stage:version-id
+  #exported-variables:
+    # - variable
+    # - variable
+  #git-credential-helper: yes
+#batch:
+  #fast-fail: true
+  #build-list:
+  #build-matrix:
+  #build-graph:
+phases:
+  install:
+    runtime-versions:
+      docker: 19
+  #install:
+    #If you use the Ubuntu standard image 2.0 or later, you must specify runtime-versions.
+    #If you specify runtime-versions and use an image other than Ubuntu standard image 2.0, the build fails.
+    #runtime-versions:
+      # name: version
+      # name: version
+    #commands:
+      # - command
+      # - command
+  pre_build:
+    commands:
+      - make docker-hub-signin
+      - echo $CODEBUILD_RESOLVED_SOURCE_VERSION
+  build:
+    commands:
+      - echo "[Running Unit Tests]"
+      - make tester-build
+      - make unit-test
+      # - command
+  #post_build:
+    #commands:
+      # - command
+      # - command
+reports:
+  unit-tests:
+    files:
+      - testresults.xml
+    base-directory: $CODEBUILD_SRC_DIR/application
+    discard-paths: yes
+    file-format: JUNITXML
+#artifacts:
```

Comment:
Once we're finished with developing this pipeline we should remove the commented out sections from build spec yml files

Reply:
I think if we're going to make those kinda of pipeline changes fairly soon, then it's ok to leave the comment out parts, but if it's more long term then it would be better to remove it. Removing the comment would make it more readable, and we can always look up the build-spec stuff in the future

That said don't mind, I'll leave it to you to decide if it's need